### ISSUE TITLE
There is maybe a mistake in models/official/vision/detection/modeling/architecture/fpn.py.

### ISSUE THREAD

**Issue by 2696120622** [tensorflow/models#7759]
# Adds coarser FPN levels introduced for RetinaNet.
      for level in range(backbone_max_level + 1, self._max_level + 1):
        feats_in = feats[level - 1]
        if level > backbone_max_level + 1:
          feats_in = tf.nn.relu(feats_in)
        feats[level] = tf.keras.layers.Conv2D(
            filters=self._fpn_feat_dims,
            strides=(2, 2),
            kernel_size=(3, 3),
            padding='same',
            name='p%d' % level)(
                feats_in)
These lines of code above maybe have a conflict with the paper.
In paper,P6 is obtained via a 3x3 stride-2 conv on C5,and P7 is computed by applying ReLU followed by a 3x3 stride-2 conv on P6.
However,in the code above,P6 is obtained via a 3x3 stride-2 conv on P5.

**Comment by yeqingli** [comment]
It is an alternative implementation. Wouldn't hurt the performance.
